core/keyserver/keyserver_test.go

```diff
@@ -139,15 +139,19 @@ func TestLatestRevision(t *testing.T) {
 					}).
 					Return(&tpb.GetMapLeavesResponse{
 						MapLeafInclusion: []*tpb.MapLeafInclusion{{}},
-					}, nil)
+					}, nil).Times(2)
 				e.s.Log.EXPECT().GetInclusionProof(gomock.Any(), gomock.Any()).
-					Return(&tpb.GetInclusionProofResponse{}, nil)
+					Return(&tpb.GetInclusionProofResponse{}, nil).Times(2)
 			}
 
 			_, err = e.srv.ListEntryHistory(ctx, &pb.ListEntryHistoryRequest{DirectoryId: directoryID})
 			if got, want := status.Code(err), tc.wantErr; got != want {
 				t.Errorf("ListEntryHistory(): %v, want %v", err, tc.wantErr)
 			}
+			_, err = e.srv.ListUserRevisions(ctx, &pb.ListUserRevisionsRequest{DirectoryId: directoryID})
```

Comment:
why not validate the response?

Reply:
There's no client code to perform validation yet - I'll add that plus an integration test in another PR.